modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= Creating multiple ingresses through single ALB
+
+You can route the traffic to multiple Ingresses through a single AWS Load Balancer (ALB).
```

Comment:
As with the earlier modules, it is unclear whether the steps in the procedure are for the web console or the CLI. I presume that it's the CLI. 
Consider updating the title or the lead-in sentence, accordingly. Consider adding a prerequisite for it. For example, Log in as a user with cluster-admin privileges.